test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -4,11 +4,11 @@
   (:require [clojure.test :refer :all]
             [clojure.java.io :refer [resource]]
             [me.raynes.fs :as fs]
-            [schema.core :as schema]
```

Comment:
Oh yeah, looks like all the ones you added have that.  Just throwing it out there - was thinking maybe the original author had intended to do that with that schema require but maybe not.